app/src/main/java/com/andrew/tasky/agenda/data/Task.kt

```diff
@@ -1,4 +1,4 @@
-package com.andrew.tasky.agenda.util
+package com.andrew.tasky.agenda.data
```

Comment:
`Task` sounds more like a domain model's name to me. In the data layer I would expect `TaskDto` or `TaskEntity` 🤔